packages/dnb-eufemia/src/extensions/forms/Value/Selection/SelectionDocs.ts

```diff
@@ -0,0 +1,9 @@
+import { PropertiesTableProps } from '../../../../shared/types'
+
+export const SelectionProperties: PropertiesTableProps = {
+  dataPath: {
+    doc: 'The path to the context data (Form.Handler). The object needs to have a `value` and a `title` property. The generated options will be placed above given JSX based children.',
```

Comment:
```suggestion
    doc: 'The path to the context data (Form.Handler). The context data object needs to have a `value` and a `title` property. The generated options will be placed above given JSX based children.',
```

When reading this in the [dos](https://eufemia-git-feat-value-selection-eufemia.vercel.app/uilib/extensions/forms/Value/Selection/properties/#field-specific-props), I don't 100% get it.